pages/community/community-support-precice.md

```diff
@@ -0,0 +1,52 @@
+
+Directly [contact Jun. Prof. Benjamin Uekermann](https://www.ipvs.uni-stuttgart.de/departments/us3/) to discuss a collaboration.
+
+### 2. Support license
+
+A support license is the easiest way to support preCICE. By acquiring a support license, you directly contribute to the development of preCICE through a fixed amount per year. In exchange, you receive a number of benefits, in particular timely and direct support from the preCICE developers around any issues that may arise when using preCICE. We provide two service levels, with discounted rates for academia.
+
+#### Direct cost in a research project
+
+As an academic user you can declare the support license, like any software license, as a direct cost/consumable in your research proposal (see e.g. section 2.1 in [basic module DFG guidelines](https://www.dfg.de/formulare/52_01/52_01_en.pdf)). It is also recommended to include the support license as a risk mitigation strategy to highlight that you, as a project lead, have the support for a successful implementation. In addition, as part of the support license, you can leverage the preCICE website and community for your communication/dissemination activities.
+
+[Speak to Jun. Prof. Benjamin Uekermann](https://www.ipvs.uni-stuttgart.de/departments/us3/) to learn more.
+
+#### Benefits
+
+|   | Basic support license <br>(12 months, billed anually)  | Extended support license <br>(12 months, billed anually)  |
+| - | - | -|
+| **discounts for preCICE Workshop** | up to 3 tickets | up to 3 tickets |
+| **priority support**   | via Discourse, up to 3 badges | via Discourse, up to 3 badges  |
+| **visibility/dissemination**   | listed as supporter, project featured on website | listed as supporter, project featured on website  |
+| **private support**   |  | via email or Zoom |
+| **on-site support**    |  | up to 2 days of training or support |
+| **service level agreement** | | upon request |
+| **price** | upon request (discount for academia) | upon request (discount for academia) |
+
+To request a support license [contact Jun. Prof. Benjamin Uekermann](https://www.ipvs.uni-stuttgart.de/departments/us3/).
+
+### 3. (Sub-) contracting
+
+Contracting is another option to benefit from the expertise of the preCICE developers. As an academic user, this would typically be as a sub-contractor for a specific work package or task in your research project, e.g. active support in the development of a new adapter.
+
+Please note that we can only offer a very limited amount of contracting and we strongly advise you to include preCICE as a project partner, or acquire a support license instead.
```

Comment:
The part of what we can actually help with and the part that there should preferably be some alignment with research (potentially with a link to the Roadmap) is missing here. Is this intentional at the moment?